packages/public-api/src/mobile.ts

```diff
@@ -1,7 +1,14 @@
+import type { JsonRpcPayload, JsonRpcResponse } from 'web3-core-helpers'
+
 /**
  * APIs that both Android and iOS implements and have the same API signature
  */
-export interface SharedNativeAPIs {}
+export interface SharedNativeAPIs {
+    /**
+     * Send Ethereum JSON RPC
+     */
+    send(payload: JsonRpcPayload): Promise<JsonRpcResponse>
```

Comment:
```suggestion
    send(payload: unknown): Promise<unknown>
```

I don't like the idea of relying on external types.